README.md

```diff
@@ -60,6 +61,11 @@ channel0 = ShadertoyChannel(image_data, wrap="repeat")
 shader = Shadertoy(shader_code, resolution=(800, 450), inputs=[channel0])
 ```
 
+To easily load shaders from the webseit make use of the `.from_id` or `.from_json` classmethods. This will also download supported input media.
```

Comment:
```suggestion
To easily load shaders from the website make use of the `.from_id` or `.from_json` classmethods. This will also download supported input media.
```